### PR TITLE
Add translation

### PR DESCRIPTION
**Pull request by Croden1999**
# About
Add Traditional and Simplified Chinese translation.

### CODE REVIEW

**Comment by bucanero** [comment]
Hello @Croden1999 , thanks for the  Chinese translation.

The app doesn't yet support UTF-8 strings for the UI  user messages, so before I merge it, some changes need to be done to the source code.

Once those changes are ready I'll add it. 👍

**Comment by Croden1999** [comment]
> Hello @Croden1999 , thanks for the Chinese translation.
> 
> The app doesn't yet support UTF-8 strings for the UI user messages, so before I merge it, some changes need to be done to the source code.
> 
> Once those changes are ready I'll add it. +1

thx a lot, sir.

File: pkgfiles/USRDIR/LANG/ch.po [hunk 1]
```diff
@@ -0,0 +1,390 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PKGi PS3\n"
+"POT-Creation-Date: 2021-10-09 19:00-0300\n"
+"PO-Revision-Date: 2021-10-09 19:00-0300\n"
+"Last-Translator: Algol\n"
```
Comment (bucanero) [inline pkgfiles/USRDIR/LANG/ch.po:6]:
```suggestion
"Last-Translator: Croden1999\n"
```

File: pkgfiles/USRDIR/LANG/zh.po [hunk 2]
```diff
@@ -0,0 +1,390 @@
+msgid ""
+msgstr ""
+"Project-Id-Version: PKGi PS3\n"
+"POT-Creation-Date: 2021-10-09 19:00-0300\n"
+"PO-Revision-Date: 2021-10-09 19:00-0300\n"
+"Last-Translator: Algol\n"
```
Comment (bucanero) [inline pkgfiles/USRDIR/LANG/zh.po:6]:
```suggestion
"Last-Translator: Croden1999\n"
```

**Review by bucanero — APPROVED** [review]

**Pull request merged**